.circleci/config.yml

```diff
@@ -414,6 +414,7 @@ jobs:
 
   profile:
     <<: *contrib_job
+    resource_class: large
```

Comment:
This looks unrelated?

Reply:
yeah somewhat, just wanted my CI to be faster 😎